server/sonar-web/src/main/js/apps/issues/components/AppContainer.tsx

```diff
@@ -67,6 +80,17 @@ const fetchIssues = (query /*: RawQuery */, requestOrganizations /*: boolean */
     .catch(throwGlobalError);
 };
 
-const mapDispatchToProps = { fetchIssues };
+interface DispatchProps {
+  fetchIssues: (query: RawQuery, requestOrganizations?: boolean) => Promise<void>;
+}
+
+// have to type cast this, because of async action
+const mapDispatchToProps = { fetchIssues: fetchIssues as any } as DispatchProps;
+
+interface OwnProps {
+  myIssues?: boolean;
```

Comment:
I would add `location` here too, and probably all other properties from `App` that are not directly defined in this container, to better document how this container should be used (like in the `ExploreIssues` for example)